homeassistant/components/pilight.py

```diff
@@ -79,6 +86,8 @@ def send_code(call):
 
         try:
             pilight_client.send_code(message_data)
+            if send_delay > 0.0:
+                time.sleep(send_delay)
```

Comment:
@balloob Might this be a working solution: https://gist.github.com/janLo/5dba266f87ed3cdc0699720212d73757 ?

I'm not used to the async programming in python.